src/components/Banner/bannerChangelogData.ts

```diff
@@ -9,6 +9,13 @@
 import { ChangelogData } from "../../utils/ComponentChangelogTable";
 
 export const changelogData: ChangelogData[] = [
+  {
+    date: "2024-07-30",
```

Comment:
Change this to "Prerelease" as well. We'll update this when the npm release goes out.